package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test:backend": "mocha test/mocha",
-    "lint": "eslint src"
+    "lint": "eslint src",
```

Comment:
this task should run tests on every `*.js` file.
this only runs in the src folder